ParentWorkspace/SpringModule/src/main/java/com/miniproject/model/Orders.java

```diff
@@ -0,0 +1,85 @@
+package com.miniproject.model;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+/**
+ * @author Mohaneswaran
+ *
+ *         maps with user table with many to one cardinality.
+ */
+@Entity
+@Table(name = "ORDERS")
+
+public class Orders {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "ORDERID")
+	private Integer orderId;
+	
+	@ManyToOne
+	@JoinColumn(name = "USERID", nullable = false)
+	private User userId;
+	
+	@Column(name = "STATUS")
+	private String status;
+
+	@Column(name = "ORDERDATE")
+	private Date orderDate;
+	
+	
+
+	public Orders(Integer orderId, User userId, String status, Date orderDate) {
```

Comment:
orderId is Primary key(generated automatically) and should not be passed as a parameter for this object.